### PR TITLE
add scala jsonapi

### PR DESCRIPTION
**Pull request by emanresusername**

### CODE REVIEW

File: implementations/index.md [hunk 1]
```diff
@@ -159,6 +159,9 @@ has a page describing how to emit conformant JSON.
   * [katharsis-servlet](https://github.com/katharsis-project/katharsis-servlet) is a generic servlet/filter adapter for [katharsis](http://katharsis.io) core module. This module can be used in traditional servlet or filter based Java web applications, or even non-Servlet-API-based web applications such as Portal/Portlet, Wicket, etc.
 * [Elide](http://elide.io) is a web framework supporting JSON API. Through annotation-based JSON API endpoint generation, Elide enables you to focus on your data model, security model, and business logic while avoiding unnecessary boilerplate. Moreover, through use of the JSON API Patch extension, [Elide](http://elide.io) provides full support for database transactions.
 
+### <a href="#server-libraries-java" id="server-libraries-java" class="headerlink"></a> Scala
```
Comment (dgeb) [inline implementations/index.md]: Could you please update the id and href to `server-libraries-scala` ?

Comment (emanresusername) [inline implementations/index.md]: haha, little too much copypasta, my bad, will do

**Comment by emanresusername** [comment]
@dgeb want me to squash all these into one commit as well?

**Comment by dgeb** [comment]
@emanresusername that would be great - thanks :)

**Comment by emanresusername** [comment]
@dgeb  gif-keyboard-13114280727587009893

**Comment by dgeb** [comment]
@emanresusername lookin' good - thanks 💯

**Pull request merged**